frontend/ui/widget/dictquicklookup.lua

```diff
@@ -832,6 +846,7 @@ function DictQuickLookup:_instantiateScrollWidget()
             auto_para_direction = not self.is_wiki, -- only for dict results (we don't know their lang)
             image_alt_face = self.image_alt_face,
             images = self.images,
+            display_highlight = true,
```

Comment:
I added that do not draw option to TextBoxWidget because it is used at so many places, and I did not want to break anything. I could easily test HtmlBoxWidget everywhere, so I did not add that option there.

But I agree, it is better to keep them in sync, so I added it to HtmlBoxWidget too, and also renamed the member to highlight_text_selection.